### PR TITLE
updated to twitter bootstrap 4.0.0

### PR DESCRIPTION
**Pull request by sergiohermes**

### CODE REVIEW

File: module/Application/view/layout/layout.phtml [hunk 1]
```diff
@@ -1,55 +1,70 @@
-<?= $this->doctype() ?>
-
+<?php echo $this->doctype() ?>
 <html lang="en">
-    <head>
-        <meta charset="utf-8">
-        <?= $this->headTitle('ZF Skeleton Application')->setSeparator(' - ')->setAutoEscape(false) ?>
+  <head>
```
Comment (robinson-j16) [inline module/Application/view/layout/layout.phtml]: Should the <head> and <body> tags be indentend by 4 spaces?

**Comment by webimpress** [comment]
@sergiohermes Can you add screenshot, please? 😄

**Comment by sergiohermes** [comment]
image

image

image

image

File: module/Application/view/layout/layout.phtml [hunk 2]
```diff
@@ -1,55 +1,71 @@
...
+        ->prependFile('https://code.jquery.com/jquery-3.3.1.min.js');
+?>
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <link rel="shortcut icon" href="https://framework.zend.com/ico/favicon.ico" />
+    <?=$this->headTitle('ZF Skeleton Application')->setSeparator(' - ')->setAutoEscape(false)?>
```
Comment (alextech) [inline module/Application/view/layout/layout.phtml]: need space between <?= and $this to be consistent with other usage.

**Comment by weierophinney** [comment]
Closing in favor of #431.

**Pull request closed without merging**